Cargo.toml

```diff
@@ -21,14 +21,15 @@ env_logger = "0.10.0"
 itertools = "^0.10"
 ts-rs = "6.2.1"
 rand = "0.8.5"
-async_zip = {version="0.0.15", features=["full"]}
+async_zip = {version="0.0.15", features=["tokio", "tokio-fs", "deflate", "bzip2", "lzma", "zstd", "xz"]}
```

Comment:
please let's try to only use "deflate" -- core-rust would fail anyway, so it shouldn't be supported i think. it also increases the "attack" surface if any of the algorithms has a security problem.   If we get the "compression" error we need to find out which algo precisely is the problem/needed.  Could you try with the "app-files.zip" i sent you what is really needed?